server/src/main/java/io/crate/planner/Planner.java

```diff
@@ -435,6 +435,12 @@ public Plan visitRestoreSnapshotAnalyzedStatement(AnalyzedRestoreSnapshot analys
     @Override
     public Plan visitSetStatement(AnalyzedSetStatement setStatement, PlannerContext context) {
         switch (setStatement.scope()) {
+            case TIME_ZONE:
+                var settings = setStatement.settings();
+                if(!settings.get(0).expression().toString().equalsIgnoreCase("'utc'")) {
```

Comment:
Would this work for parameter symbols or similar - like `SET TIME ZONE ?` ?

Reply:
The is grammar currently defines as `SET TIME ZONE (LOCAL | DEFAULT | stringLiteral)` so a parameter symbol is not allowed here.